api/server/router/image/backend.go

```diff
@@ -0,0 +1,29 @@
+package image
+
+import (
+	"io"
+
+	"github.com/docker/distribution/reference"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/builder/dockerfile"
```

Comment:
@calavera i just realized, it probably would be better to have a separate router package for `build`, since it will provide more granularity and not force people to import `builder/dockerfile` just because they want to import the api.